src/box/index.h

```diff
@@ -751,7 +751,7 @@ struct index_read_view_iterator_base {
 };
 
 /** Size of the index_read_view_iterator struct. */
-#define INDEX_READ_VIEW_ITERATOR_SIZE 80
+#define INDEX_READ_VIEW_ITERATOR_SIZE 88
```

Comment:
Shouldn't be necessary if https://github.com/tarantool/tarantool-ee/pull/955#discussion_r1824534000 is addressed.

Reply:
Thanx! Closed.